TourMate/TourMate/Frontend/Plan/Views/CommentViews/CommentView/CommentInteractionView.swift

```diff
@@ -42,14 +42,12 @@ struct CommentInteractionView: View {
                 .background(.white)
                 .cornerRadius(20.0)
             }
+            .allowsHitTesting(viewModel.allowUserInteraction)
```

Comment:
Do we need both the check in the if-statement and the `allowsHitTesting` part? Seems like one is enough